### PR TITLE
set up

### LINKED ISSUE #1: Project setup
# REQUIREMENTS
Initial setup with react native CLI 

# DESIGN
N/A

# Acceptance Criteria
List all required acceptance criteria
- [ ] Initial screen "Hello World" text

### PR DESCRIPTION
**Pull request by alcamk**
Co-Authored-By: alcamk <alvarezohc@gmail.com>

# Description
Initial setup configuration 
 
 # Testing
Clone repository 
Run in terminal $cd "folder/direction" 
Run $npm install
Run $npx react-native start
If you are running for android run $npx react-native run-android 
For ios run $npx react-native run-ios

  ## Screenshots 
n/a

### CODE REVIEW

**Review by Ing-Heber — DISMISSED** [review]
Nice Job team, the app worked!!

**Review by tovl1 — DISMISSED** [review]

**Review by OwenRamire — APPROVED** [review]
Nice 🚀  in the next commits, try to be more specific about the things you did. I recommend this blog about [meaningful commits](https://medium.com/@menuka/writing-meaningful-git-commit-messages-a62756b65c81)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+tribomobile/node_modules
```
Comment (fernandoamz) [inline .gitignore:1]: new blank line

File: tribomobile/App.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ * @flow strict-local
+ */
```
Comment (fernandoamz) [inline tribomobile/App.js:7]:
```suggestion
```

Try to don't use comments on the files

File: tribomobile/App.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  StatusBar,
+} from 'react-native';
+
+
+const App: () => React$Node = () => {
+  return (
+    <>
+      <StatusBar barStyle="dark-content" />
+      <SafeAreaView>
+        <Text>Hola Tribo App</Text>
```
Comment (fernandoamz) [inline tribomobile/App.js:22]: Add the text to a constant file

File: tribomobile/__tests__/App-test.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/**
+ * @format
+ */
```
Comment (fernandoamz) [inline tribomobile/__tests__/App-test.js:3]:
```suggestion
```

File: tribomobile/__tests__/App-test.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+import 'react-native';
+import React from 'react';
+import App from '../App';
+
+// Note: test renderer must be required after react-native.
```
Comment (fernandoamz) [inline tribomobile/__tests__/App-test.js:9]:
```suggestion
```

**Review by fernandoamz — APPROVED** [review]

**Review by luisarmandodv — APPROVED** [review]

**Review by JordiGN — APPROVED** [review]

**Review by Ing-Heber — APPROVED** [review]

**Pull request merged**